custom_components/opnsense/coordinator.py

```
@@ -119,144 +106,146 @@
+                        # Kbs = KBs * 8
+                        value = KBs
+
+                    new_property = f"{prop_name}_{label}"
+                    interface[new_property] = int(round(value, 0))
+
+            for server_name in dict_get(
+                self._state, "telemetry.openvpn.servers", {}
+            ).keys():
+                if (
+                    server_name
+                    not in dict_get(self._state, "telemetry.openvpn.servers", {}).keys()
```

Ruff

Use `key not in dict` instead of `key not in dict.keys()`

Remove `.keys()`

(SIM118)